testapps/testapp/src/main/res/raw/msal_config.json

```diff
@@ -1,7 +1,7 @@
 {
-  "client_id" : "9851987a-55e5-46e2-8d70-75f8dc060f21",
+  "client_id" : "b92e0ba5-f86e-4411-8e18-6b5f928d968a",
```

Comment:
9851987a-55e5-46e2-8d70-75f8dc060f21  is not lab app client id. Going to use 4b0db8c2-9f26-4417-8bde-3f0e3656f8e0 instead.